docs/source/command_line.rst

```diff
@@ -1132,6 +1132,14 @@ List of currently incomplete/experimental features:
          # Without PreciseTupleTypes: tuple[int, ...]
          # With PreciseTupleTypes: tuple[()] | tuple[int] | tuple[int, int]
 
+* ``NewInlineTypedDict``: this feature enables :pep:`764` syntax for inline
+  :ref:`TypedDicts <typeddict>`, for example:
+
+  .. code-block:: python
+
+     def test_values() -> TypedDict[{"int": int, "str": str}]:
+         return {"int": 42, "str": "test"}
```

Comment:
Maybe we can use different field names to avoid the confusion with the actual types?

```suggestion
     def test_values() -> TypedDict[{"test_int": int, "test_str": str}]:
         return {"test_int": 42, "test_str": "test"}
```

or similar.

Edit: I know see that this was taken from an already existing example. Feel free to apply/ignore.